### DISCUSSION TITLE
How to implement ERC-1155 standard on Aptos.=

### DISCUSSION THREAD

**Discussion by frey-eth** [aptos-labs/aptos-developer-discussions#530] [Questions]
### Discord user ID

frey9846

### Describe your question in detail.

Hi everyone,

I am exploring how to implement the ERC-1155 token standard on the Aptos platform. I would like to know if there is a way to implement this standard on Aptos using Move VM. If so, does anyone have experience or detailed guides on how to develop and deploy ERC-1155 tokens on Aptos?

Thanks in advance for your help!

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

Aptos CLI

**Comment by yue1823** [comment]
I am interested in your plans,if you would want to discuss them further, please contact me in DC.  

Dc `._yue._.`

**Comment by ying-w** [comment]
https://aptos.dev/en/build/smart-contracts/fungible-asset
